test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -74,6 +76,10 @@
     private static final String PRAVEGA_BOOKKEEPER_VERSION = System.getProperty("pravegaBookkeeperVersion", PRAVEGA_VERSION);
     private static final String PRAVEGA_OPERATOR_VERSION = System.getProperty("pravegaOperatorVersion", "latest");
     private static final String PREFIX = System.getProperty("imagePrefix", "pravega");
+    private static final String TIER2_NFS = "nfs";
+    private static final String TIER2_ECS = "ecs";
```

Comment:
I'm wondering whether we really need to create these constants. For one thing, imagine that in the future we implement new bindings. We would have to remember to come back here and create a new constant. Granted that we won't do it often.

From the code below, it looks like you are only doing something special for `NFS`, so could only distinguish between `NFS` and others?

Reply:
yes, it is not required. Refactored the code.